ingest/entryReader.go

```diff
@@ -286,6 +287,9 @@ func (er *EntryReader) read() (*entry.Entry, error) {
 	if err = er.fillHeader(ent, &id, &sz); err != nil {
 		return nil, err
 	}
+	if sz > uint32(MAX_ENTRY_SIZE) {
```

Comment:
this is already checked at the end of fillHeader

Reply:
shit, fixing